modules/user-provided-certificates-for-api-server.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// *authentication/certificate-types-descriptions.adoc
+
+[id="user-provided-certificates-for-the-api-server_{context}"]
+= User-provided certificates for the API server
+
+[discrete]
+== Purpose
+
+The API server is accessible by clients external to the cluster at
+`api.<cluster_name>.<base_domain>`. You might want clients to access the API
+server at a different host name or without the need to distribute the
+cluster-managed certificate authority (CA) certificates to the clients. The
+administrator must set a custom default certificate to be used by the API server
+when serving content.
+
+[discrete]
+== Location
+
+The user-provided certificates must be provided in a `kubernetes.io/tls` type
+`Secret` in the `openshift-config` namespace. Update the API server cluster
+configuration, the `apiserver/cluster` resource, to enable the use of the
+user-provided certificate.
+
+[discrete]
+== Expiration
+
+User-provided certificates are managed by the user.
+
+[discrete]
+== Customization
+
+Update the secret containing the user-managed certificate as needed.
```

Comment:
Better to link https://docs.openshift.com/container-platform/4.3/authentication/certificates/api-server.html as "how to"?

Reply:
Unfortunately with our current tooling, I cannot link a module to another module. However, I added that as a link in the Additional resources section at the bottom of the document https://github.com/openshift/openshift-docs/pull/18254/files#diff-56d951a231462af6d950dac7d86eec5cR33